tests/ts/liblastlog2/dlopen

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+TS_TOPDIR="${0%/*}/../.."
+TS_DESC="dlopen"
+
+. "$TS_TOPDIR"/functions.sh
+ts_init "$*"
+
+TESTPROG="$TS_HELPER_LIBLASTLOG2_DELOPEN"
+
+[ -x $TESTPROG ] || ts_skip "test not compiled"
+
+ts_run $TESTPROG >> $TS_OUTPUT.debug 2>&1
```

Comment:
This testprog needs to be passed the `.so` as argument to actually do something.